src/TodoList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import TodoListItem from "./TodoListItem";
+
+var todoList = [
+  {
+    id: 1,
+    title: "Assignment not start",
+  },
+  {
+    id: 2,
+    title: "Assignment start",
+  },
+  {
+    id: 3,
+    title: "Assignment completed",
+  },
+];
```

Comment:
the `var` keyword is old JS syntax and can cause a lot of issues and bugs if not managed properly. For good practice, it's always better to use either `let` or `const` when declaring variables.